lib/takeaway.rb

```diff
@@ -0,0 +1,13 @@
+class Takeaway
+
+  def initialize(menu, order, sms)
```

Comment:
If you do decide that we need this class, the `order`  and `sms` inputs here are not currently being used. How might using this variable help us use dependency injection?

https://github.com/makersacademy/skills-workshops/blob/f5b4801840fe07d26ff70341652dc81dcda12289/practicals/object_oriented_design/dependency_injection.md might be useful 🙂